snap/config/security-secret-store/pki-setup-config-kong.env

```diff
@@ -0,0 +1,55 @@
+#  ----------------------------------------------------------------------------------
+#  pki-setup-config-kong.env	version 1.0 created June 13, 2018
```

Comment:
Same comment as previous about using sed instead of duplicating the entire file.

Reply:
This is done now